README.md

```diff
@@ -77,11 +77,10 @@ Apart from MMDetection, we also released a library [mmcv](https://github.com/ope
 
 ### 💎 Stable version
 
-**2.28.0** was released in 28/1/2023:
+**2.28.1** was released in 1/2/2023:
 
-- Support Objects365 Dataset, and Separated and Occluded COCO metric
-- Support acceleration of RetinaNet and SSD on Ascend
-- Deprecate the support of Python 3.6
+- Fix import error that causes training failure.
```

Comment:
Consider still use the the highlights of 2.28.0 and indicate it fix small issues of 2.28.0 might be better.